stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewGroupDescriptor.java

```diff
@@ -111,11 +87,13 @@ public void markDecorView() {
       mIsDecorView = true;
     }
 
-    public int getChildCount() {
-      if (mIsDecorView) {
-        return getDecorViewChildCount();
-      } else {
-        return mElement.getChildCount();
+    public void getChildren(Accumulator<Object> children) {
+      for (int i = 0, N = mElement.getChildCount(); i < N; ++i) {
+        final View child = mElement.getChildAt(i);
+        if (isChildVisible(child)) {
```

Comment:
This can be answered by looking at isChildVisible() :) isChildVisible()'s job is mostly to hide DOMHiddenViews (e.g. highlighting stuff). This code isn't semantically changed from before but because of the change in protocol (getChildCount/At vs. getChildren), it's much simpler.